push_ghcr.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+echo $GITHUB_TOKEN | docker login ghcr.io --username $GITHUB_USERNAME --password-stdin
+docker tag $timestamp_holder ghcr.io/$GITHUB_USERNAME/$timestamp_holder
```

Comment:
Have you tied to see what would be the eventual name of the pushed image? The URL now seems to contain Github username and `nsls2` user name as a part of the `timestamp_holder`.

Will it look like this - https://ghcr.io/johngedell/nsls2/nsls2-analysis-2021-1.2:latest? I think we don't need the `/nsls2/` piece here.

Reply:
Okay. I fixed this.